.evergreen.yml

```
@@ -75,6 +75,7 @@ functions:
 
           export NODE_JS_VERSION="12.4.0"
           export NPM_VERSION="7"
+          export NPM_CONFIG_CACHE="${NPM_CACHE_DIR}"
```

[npm docs](https://docs.npmjs.com/cli/v7/using-npm/config#environment-variables) say that during script runs npm will set its own env variables and although env vars are case-insensitive, if both are set, npm will prefer lower case ones. So should we use lower case just in case?

```suggestion
          export npm_config_cache="${NPM_CACHE_DIR}"
```

---

Yep